ecosystem/sep-0010.md

```diff
@@ -19,8 +19,8 @@ This SEP defines the standard way for clients such as wallets or exchanges to cr
 This protocol is a variation of mutual challenge-response, which uses Stellar transactions to encode challenges and responses.
 
 It involves three components:
-- A domain hosting a [SEP-1 stellar.toml](sep-0001.md) containing a `WEB_AUTH_ENDPOINT` and `SIGNING_KEY`, referred to as the home domain.
-- An endpoint providing the GET and POST operations discussed in this document.
+- A domain hosting a [SEP-1 stellar.toml](sep-0001.md) containing a `WEB_AUTH_ENDPOINT` (URL) and `SIGNING_KEY` (`G...`), referred to as the home domain.
+- An endpoint providing the GET and POST operations discussed in this document. The endpoint may be hosted on the home domain, a sub-domain of the home domain, or any other domain. The domain that it is hosted is referred to as the web auth domain. 
```

Comment:
This last sentence doesn't read well, `The domain that it is hosted`?

Reply:
```suggestion
- An endpoint providing the GET and POST operations discussed in this document. The endpoint may be hosted on the home domain, a sub-domain of the home domain, or any other domain. The domain that the endpoint is hosted is referred to as the web auth domain. 
```

Reply:
Added `on`, LGTM now